Raygun.Druid4Net.IntegrationTests/Raygun.Druid4Net.IntegrationTests.csproj

```diff
@@ -1,90 +1,30 @@
-    </Reference>
-    <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed">
-      <HintPath>..\packages\Newtonsoft.Json.12.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
-      <Private>True</Private>
-    </Reference>
-    <Reference Include="nunit.framework, Version=3.9.0.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
-      <HintPath>..\packages\NUnit.3.9.0\lib\net45\nunit.framework.dll</HintPath>
-    </Reference>
-    <Reference Include="Sigil, Version=4.7.0.0, Culture=neutral, PublicKeyToken=2d06c3494341c8ab, processorArchitecture=MSIL">
-      <HintPath>..\packages\Sigil.4.7.0\lib\net45\Sigil.dll</HintPath>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Configuration" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Runtime.Serialization" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="JilSerializer.cs" />
-    <Compile Include="NewtonsoftSerializer.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Queries\GroupBy\CountriesWithLimitAndOrderBy.cs" />
-    <Compile Include="Queries\GroupBy\PagesGroupedByCity.cs" />
-    <Compile Include="Queries\Scan\ScanFilteredData.cs" />
-    <Compile Include="Queries\Scan\ScanVirtualColumns.cs" />
-    <Compile Include="Queries\Search\BasicSearch5Pages.cs" />
-    <Compile Include="Queries\SegmentMetadata\BasicSegmentMetadata.cs" />
-    <Compile Include="Queries\TestQueryBase.cs" />
-    <Compile Include="Queries\TimeBoundary\FilteredTimeBoundary.cs" />
-    <Compile Include="Queries\Timeseries\PagesAddedOverTimeByHour.cs" />
-    <Compile Include="Queries\TopN\AsyncBasicTop10Pages.cs" />
-    <Compile Include="Queries\TopN\FilteredTop5Pages.cs" />
-    <Compile Include="Queries\TopN\BasicTop10Pages.cs" />
-    <Compile Include="Queries\Wikipedia.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Raygun.Druid4Net\Raygun.Druid4Net.csproj">
-      <Project>{a0971ac2-69bd-4e48-882a-9cf6a9253d3a}</Project>
-      <Name>Raygun.Druid4Net</Name>
-    </ProjectReference>
-  </ItemGroup>
   <ItemGroup>
-    <None Include="App.config" />
-    <None Include="packages.config" />
+    <ProjectReference Include="..\Raygun.Druid4Net\Raygun.Druid4Net.csproj" />
   </ItemGroup>
   <ItemGroup>
     <Content Include="wikipedia-ingestion-spec.json" />
     <Content Include="wikipedia.json.gz" />
+    <Content Include="appsettings.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Jil" Version="2.15.4" />
```

Comment:
This package and `sigil` do not appear to be maintained anymore. I think you should look at removing them